.github/workflows/test.yml

```diff
@@ -8,8 +8,21 @@ env:
   GO111MODULE: on
 
 jobs:
-
-
+  golangci:
```

Comment:
Please, keep the style consistent in the YAML file:

- Blank line before and after the job, and before and after steps.
- No need to add a `name` field to the `uses` steps, which are self-explaining by the action name.